qiskit_nature/transformers/second_quantization/freeze_core_transformer.py

```diff
@@ -14,14 +14,17 @@
 
 from typing import List, Optional
 
-from qiskit_nature.drivers.second_quantization import QMolecule
+from qiskit_nature.deprecation import DeprecatedType, warn_deprecated_same_type_name
+from qiskit_nature.properties.second_quantization.electronic import ElectronicDriverResult
 
 from .active_space_transformer import ActiveSpaceTransformer
+from .electronic import FreezeCoreTransformer as NewFreezeCoreTransformer
 
 
 class FreezeCoreTransformer(ActiveSpaceTransformer):
-    """The Freeze-Core reduction."""
+    """**DEPRECATED!**"""
```

Comment:
Actually, the `qiskit_nature.transformers.second_quantization` module has not been released yet. Thus, after we merge #283 I can update the deprecation of `qiskit_nature.transformers.*` to point directly to the `electronic` module and remove these files entirely.